libs/sdk-ui-ext/src/internal/dashboardEmbedding/ScheduledMail/utils/validate.ts

```diff
@@ -0,0 +1,8 @@
+// (C) 2019-2020 GoodData Corporation
+
+// eslint-disable-next-line no-useless-escape
+const EMAIL_REGEX = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+export function isEmail(value: string): boolean {
```

Comment:
I know it is not your code, but we should really use a library for this...

Reply:
I was afraid to touch it, I think it must have some reason why it's done like this: D

Reply:
I understand :D